gsoc/settings.py

```diff
@@ -394,3 +394,7 @@
 }
 
 TEXT_ADDITIONAL_TAGS = ('iframe',)
+
+BOT_NICK = "CommandBot"
+IRC_SERVER = "irc.freenode.org"
+RECEIVER = "limnoria"
```

Comment:
what is this for?

Reply:
This is for configuring the bot nickname, the irc server and the receiver of the commands.